src/MassTransit/DependencyInjection/Configuration/ServiceCollectionBusConfigurator.cs

```diff
@@ -26,6 +28,9 @@ IBusRegistrationContext CreateRegistrationContext(IServiceProvider provider)
             collection.AddSingleton(provider => Bind<IBus>.Create(CreateRegistrationContext(provider)));
             collection.AddSingleton(provider => provider.GetRequiredService<Bind<IBus, IBusRegistrationContext>>().Value);
 
+            collection.AddOptions();
```

Comment:
Why here? No other options are added here. Consider moving these lines to where the other options are added.

Reply:
Are  you even reading those options anywhere? And it's important to know that options will _always_ resolve, so knowing if they've actually been configured is important so that they default to `_isConfigured = false` to avoid any overhead until opted-into.